examples/android_local_test/bazel/robolectric/BUILD

```diff
@@ -0,0 +1,32 @@
+load("@rules_jvm_external//:defs.bzl", "artifact")
```

Comment:
Pending thinking about my other comment, I wonder if this file should go somewhere more discoverable as an official recommendation of how to use Bazel and robolectric. Or maybe this is a sufficiently official location and should just be pointed to from the docs for discoverability

Reply:
100%. Will think about this when we're refreshing the docs to include this project.